src/functions/Get-SkipRemainingOnFailurePlugin.ps1

```diff
@@ -29,41 +54,18 @@ function Get-SkipRemainingOnFailurePlugin {
     $p.Start = {
         param ($Context)
 
+        # TODO: Use $Context.GlobalPluginData.SkipRemainingOnFailure.SkippedCount when exposed in $Context
```

Comment:
Is this something you will address in this PR or there should be issue created for this?

Reply:
New issue. Didn't want to touch plugin interface as it wasn't critical.

I'm happy to provide a follow-up PR when name and steps to expose it in (all?) is decided.